README.md

```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
 Call for Contributions
 ----------------------
 
-NumPy appreciates help from a wide range of different backgrounds.
-Work such as high level documentation or website improvements are valuable
-and we would like to grow our team with people filling these roles.
-Small improvements or fixes are always appreciated and issues labeled as easy
-may be a good starting point.
-If you are considering larger contributions outside the traditional coding work,
-please contact us through the mailing list.
+The NumPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated. (Issues labeled as easy may be a good starting point.) If you are considering larger contributions to the source code, please contact us through the [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
+
+Writing code isn’t the only way to contribute to NumPy. You can also: 
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve our website numpy.org
+- develop graphic design for our brand assets and promotional materials
+- translate website content
+- serve as a community coordinator
+- write grant proposals and help with other fundraising efforts
+
+If you’re unsure where to start or how your skills fit in, reach out! You can ask on the mailing list or here, on GitHub, by opening a new issue or leaving a comment on a relevant issue that is already open.
+
+Those are our preferred channels (open source is open by nature), but if you’d like to speak to us in private first, contact our community coordinators at numpy-team@googlegroups.com or on Slack (write numpy-team@googlegroups.com for an invite).
+
+We also have a biweekly community call, details of which are announced on the mailing list. You are very welcome to join. 
+
+If you are new to contributing to open source, we highly recommend reading [this guide](https://opensource.guide/how-to-contribute/).
```

Comment:
formatting, and switch "we highly recommend" to a ten word summary of what they will find at the link
```suggestion
We also have a biweekly community call, details of which are announced on the
mailing list. You are very welcome to join. 

If you are new to contributing to open source, [this
guide](https://opensource.guide/how-to-contribute/) helps explain why, what,
and how to successfully get involved.
```